client/src/components/organization/index.js

```diff
@@ -22,75 +22,136 @@ import { orgPage } from '../../constants/viewStyles'
+                <p>{city}, {state}</p>
+                <p>{country} {postalCode}</p>
+                <p>{addressLine2}</p>
+              </div>
+            </div>
+            <div className="organizationUrl">
+              <div>
+                <LanguageIcon style={orgPage.urlIcon} />
+              </div>
+              <div>
+                <p>{url}</p>
+              </div>
+            </div>
           </div>
-        </div>
-      </div>
-      <div className="organizationDescription">
-        <Divider />
-        <p>Description</p>
-        {hasVideo && (
-          <div>
-            {
-              // eslint-disable-next-line
-            } <video /> {/* TODO: fix linting error */}
+          <div className="organizationDescription">
+            <Divider />
+            <p>{description}</p>
+            {hasVideo && (
+              <iframe
+                className="organizationVideo"
+                title="Organization Video"
+                allow="encrypted-media"
+                src={videoUrl}
+              />
+            )}
+            <Divider />
           </div>
-        )}
-        <Divider />
-      </div>
-      <div className="organizationEmployees">
-        <Table>
-          <TableHeader adjustForCheckbox={false} displaySelectAll={false}>
-            <TableRow>
-              <TableHeaderColumn style={orgPage.tableHeaderCol} />
-              <TableHeaderColumn>Employee</TableHeaderColumn>
-              <TableHeaderColumn>Job Title</TableHeaderColumn>
-            </TableRow>
-          </TableHeader>
-          <TableBody showRowHover displayRowCheckbox={false}>
-            <TableRow className="organizationTableRow" hoverable>
-              <TableRowColumn style={orgPage.tableRowColAvatar}>
-                <Avatar size={32} src={portraitImg} />
-              </TableRowColumn>
-              <TableRowColumn>Fred Henderson</TableRowColumn>
-              <TableRowColumn>Engineer</TableRowColumn>
-            </TableRow>
-          </TableBody>
-        </Table>
+          <div className="organizationEmployees">
+            <Table>
+              <TableHeader adjustForCheckbox={false} displaySelectAll={false}>
+                <TableRow>
+                  <TableHeaderColumn style={orgPage.tableHeaderCol} />
+                  <TableHeaderColumn>Employee</TableHeaderColumn>
+                  <TableHeaderColumn>Job Title</TableHeaderColumn>
+                </TableRow>
+              </TableHeader>
+              <TableBody showRowHover displayRowCheckbox={false}>
+                {createEmployeeTable(users)}
+              </TableBody>
+            </Table>
+          </div>
+        </div>
       </div>
-    </div>
-  </div>
-)
+    )
+  }
+}
+
+function createEmployeeTable(employees) {
+  return employees.map(employee => {
+    const {
+      id,
+      firstName,
+      lastName,
+      mainTitle,
+      media
+    } = employee
+
+    var imageUrl = null;
+
+    if (media && media[0])
```

Comment:
Use the same "reducedMedia" function used in the organization reducer to get the url of the image. I'm not sure if it's safe to assume that the image is always in index 0.